extensions/ssh-manager/src/create.tsx

```diff
@@ -0,0 +1,37 @@
+import { ActionPanel, Form, showHUD, SubmitFormAction, useNavigation } from "@raycast/api";
```

Comment:
```suggestion
import { ActionPanel, Form, showHUD, SubmitFormAction, useNavigation, randomId } from "@raycast/api";
```
You could use randomId from Raycast API instead